src/main/java/uk/gov/justice/probation/courtcasematcher/messaging/model/commonplatform/CPDefendant.java

```diff
@@ -40,7 +40,7 @@ public class CPDefendant {
     private final CPLegalEntityDefendant legalEntityDefendant;
     @JsonProperty(value="isYouth")
     private final boolean isYouth;
-    private final String cprDefendantId;
+    private final String cprUUID;
```

Comment:
just double checking, will the params above this still be returned from CPR? e.g. `isYouth`

Reply:
and will this still be considered a CPDefendant (common platform defendant) rather than a CPRDefendant?

Reply:
oh i found it, just needed to keep on reading the code `mapCprDefendantToDefendant`